tools/install-vscode-settings.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+# This script allows a user to install repo-recommended VS Code settings.
+# See: p1HpG7-sQE-p2
+#
+# Exit codes:
+#   0: All is well
+#   1: Repo settings template is missing
+#   2: Settings file creation failed
+
+# Go to monorepo root.
+cd "$(dirname "${BASH_SOURCE[0]}")/.."
+
+# Abort if settings file already exists.
+if [[ -f .vscode/settings.json ]]; then
+	echo "Settings file exists; aborting."
+	exit
+fi
+
+# Abort if repo settings file is missing.
+if [[ ! -f .vscode/settings.dist.json ]]; then
+	echo "Repo settings template is missing; aborting."
+	exit 1
+fi
+
+# Create symlink.
+cp .vscode/settings.dist.json .vscode/settings.json
```

Comment:
Should this be `ln` instead of `cp`?

Reply:
That way, you will always have updated version of settings from the repo. Otherwise, settings may get out of sync.

Reply:
We discussed this a bit as a team (p1HpG7-sQE-p2#comment-72168); using a symlink ultimately results in some of the same issues. Essentially when someone makes a change to the settings (either in the current setup or with a symlink), it inadvertently ends up in the `git` diff. That happened as recently as 12 hours ago (#37505), and also inspired your PR at #37640 (caused by #37418).

I suppose I could adapt the script to check for changes and offer to copy them...but then we'd need a way to differentiate between "managed" settings and settings customized on purpose. Thoughts?

Reply:
The way it works in Calpso is that if someone wants to manage their own settings, they can `cp` the file and then the script won't create a symlink if that file already exists.

It's just a thought. It's fine both ways though.

Reply:
Unfortunately, what has happened in the examples above is that people don't realize the settings are managed, so edited the file and committed changes unknowingly. I think I've got a decent solution now if you want to take a look.

Reply:
Great. Thanks.